doc/sphinx/proof-engine/ltac.rst

```diff
@@ -178,7 +178,7 @@ Sequence
 A sequence is an expression of the following form:
 
 .. tacn:: @expr ; @expr
-   :name: ;
+   :name: ltac-seq
```

Comment:
Can't you escape the `;` character instead? With this renaming, this index becomes completely useless.

Reply:
Ah, good point.  The problem is that this is already super brittle (the `name` tags are used as HTML anchors, and `;` can't be in one, so the cleanup / normalization process turns this into an empty id.

We could use a separate `:index:` option.

Reply:
OK, I'm happy to work on this. Will keep you posted.

Reply:
In the meantime, I would be OK to merge this and just removing this index.